### ISSUE TITLE
Diverging / poorly matched neural rates at simulation time

### ISSUE THREAD

**Issue by BalzaniEdoardo** [flatironinstitute/generalized-linear-models#46]
MLE or weakly penalized estimators for GLMs are known to perform poorly when generating new spike trains recurrently. 

If we want yo provide tutorial on fitting and simulating spikes we should keep this in mind. Here a paper that has done some interesting work on the problem:

https://proceedings.neurips.cc/paper_files/paper/2020/file/186b690e29892f137b4c34cfa40a3a4d-Paper.pdf

**Comment by BalzaniEdoardo** [comment]
Discussed the issue with Memming:

GLM not well behaved in simulaiton, especially when fitted as 1-step ahead prediction. He doesn't have a simple solution for that: 
1) the neurips above doesn't have guarantees and computationally expansive.
2) my former colleague David has another with Memming on the topic [paper](https://ieeexplore.ieee.org/document/8008426), the method is also computationally expensive.

Generally, Memming's advice is to avoid using GLMs for forecasting

**Comment by ahwillia** [comment]
I agree with Memming's take. Maybe we could even remove the simulate
feature.
